classes/submission/form/SubmissionSubmitStep1Form.php

```diff
@@ -127,6 +128,28 @@ public function setSubmissionData($submission)
         $submission->setSeriesPosition($this->getData('seriesPosition'));
         parent::setSubmissionData($submission);
     }
+
+    /**
+     * Save changes to submission.
+     * @return int the submission ID
+     */
+	function execute(...$functionParams) {
+		
+		$submissionId = parent::execute(...$functionParams);
+
+		$request = Application::get()->getRequest();
+
+		$seriesId = (int)$request->getUserVar('seriesId');
```

Comment:
@touhidurabir, this could use some validation in order to ensure that the supplied `seriesId` belongs to the current press. (I know the old code didn't do this but it's good to fix when you see it.) Something like:
```php
$seriesDao = DAORegistry::getDAO('SeriesDAO');
$series = $seriesDao->getById($request->getUserVar('seriesId'), $request->getContext()->getId());
```
Then later, rather than `$seriesId`, use: `$series ? $series->getId() : null`

Reply:
@asmecher that check has been placed but we sill need some `? :` checked as `NULL SAFE OPERATOR (?->)` not available prior to `PHP8.0+`.